media-sound/spotube-bin/spotube-bin-5.1.0.ebuild

```
@@ -0,0 +1,45 @@
+# Copyright 2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit desktop xdg
+DESCRIPTION="A lightweight Spotify client using YouTube as audio source"
+HOMEPAGE="https://github.com/KRTirtho/spotube"
+SRC_URI="https://github.com/KRTirtho/spotube/releases/download/v${PV}/spotube-linux-${PV}-x86_64.tar.xz"
+
+S="${WORKDIR}"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+
+BDEPEND="
+		dev-util/patchelf
+"
+RDEPEND="
+		>=media-video/mpv-0.38.0-r1
+		>=dev-libs/libayatana-appindicator-0.5.92
+		>=net-libs/libsoup-3.4.4
+		>=net-libs/webkit-gtk-2.48.5-r410
```

```suggestion
		media-video/mpv
		dev-libs/libayatana-appindicator
		net-libs/libsoup
		net-libs/webkit-gtk:4.1
```
看起来只有 webkit-gtk 需要指定 slot ，其他都不需要限制版本号？

---

webkit-gtk在gentoo里的slot问题的确很操蛋，gnome和很多二进制包就是只认一个特定的版本号。我的系统里已经有带三个webkit-gtk了，webkit还是指定一下版本号吧。